Codeinwp/neve
# add see all features link

### Summary
Add see all features link and changed UTM parameters for the "Get Neve Pro Now" button

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Make sure that clicking the links goes to the appropriate section when themeisle.com page is load


Closes codeinwp/neve-pro-addon#1776
